explorer/Explorer.scss

```diff
@@ -5,6 +5,11 @@ $explorer-min-width-first-col: 200px;
 $explorer-padding: 0.5rem;
 
 #ExplorerContainer {
+    @include content-wrapper;
+    @include lg-up {
+        max-width: $text-max-content-width + $graph-max-content-width + 3 *
+            $padding-x-md;
+    }
```

Comment:
We need to reset this `max-width` in iframes. Given that it's already being reset for the `.Explorer` class, I wonder what the purpose is to have the `#ExplorerContainer`?

Maybe there is some purpose and the safest option is to just add two lines to reset the `max-width` in this selector: https://github.com/owid/owid-grapher/blob/3f245756aa086df6e85f94a9e79892b37eb0381a/explorer/Explorer.scss#L19-L25

I noticed this by embedding the explorer on jsfiddle:

Live:
<img width="1231" alt="Screenshot 2022-07-05 at 09 25 21" src="https://user-images.githubusercontent.com/1308115/177294010-9a4fe5b4-5a13-4cb0-aeb1-6deac86547ef.png">

This PR:
<img width="1231" alt="Screenshot 2022-07-05 at 09 25 24" src="https://user-images.githubusercontent.com/1308115/177294039-b2789a21-ff3c-4855-b28d-dd2de36464f2.png">

 The Grapher doesn't have this issue, only Explorers are affected.

Reply:
@danielgavrilov ... and we're back!